youtube_dl/extractor/cozytv.py

```diff
@@ -0,0 +1,95 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import unified_timestamp, unified_strdate
+
+
+class CozyTVReplayIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?cozy\.tv/(?P<user>[^/]+)/replays/(?P<id>[^/]+)'
```

Comment:
`(?P<id>[^/]+)` could capture `#anchor` or `?param=value` after the `id`. Is there a more specific pattern that would work, say `[\w-]+` (based on the test URLs)?

Reply:
Although I think you may have taken "specific" more literally than I intended!